src/components/scrollManager.js

```diff
@@ -158,11 +158,22 @@ define(["dom", "browser", "layoutManager"], function (dom, browser, layoutManage
      */
     function getScrollableParent(element, vertical) {
         if (element) {
+            var nameScroll = "scrollWidth";
+            var nameClient = "clientWidth";
+            var nameClass = "scrollX";
+
+            if (vertical) {
+                nameScroll = "scrollHeight";
+                nameClient = "clientHeight";
+                nameClass = "scrollY";
+            }
+
             var parent = element.parentElement;
 
             while (parent) {
-                if ((!vertical && parent.scrollWidth > parent.clientWidth && parent.classList.contains("scrollX")) ||
-                    (vertical && parent.scrollHeight > parent.clientHeight && parent.classList.contains("scrollY"))) {
```

Comment:
why did you drop the `parent.scrollThing > parent.clientThing` check?

Reply:
I extracted their names to make a single line condition.

Reply:
Oh, you sly JavaScript lol xD